app/Http/Middleware/LanguageMiddleware.php

```diff
@@ -47,7 +48,11 @@ public function __construct(Application $app, Repository $config)
      */
     public function handle(Request $request, Closure $next)
     {
-        $this->app->setLocale($this->config->get('app.locale', 'en'));
+        if (! Auth::check() || $this->config->get('pterodactyl.lang.global')) {
```

Comment:
This should avoid using the Auth facade (use `$request->user()`).

Also, theres no reason to have the global language setting. Just always allow the user to select a language, and if one is not selected or it doesn't exist, fallback to the default language setting.

Reply:
Ok, will avoid the facade.

For the option to have a global language, I added that as I could imagine some companies would want to enable it. For example for a support matter it might be hard to give instructions in another language. However I can see how it can be unnecessary. Let me know if I should remove it or not.

Reply:
I would remove it, just have the default language be used if the client selects an invalid language.

Worst case the company can just disable this middleware.

Reply:
Yeah, I already did. It should be ready to merge.